qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -173,7 +173,8 @@ def study_prep_get_req(study_id, user_id):
                 info['start_artifact_id'] = start_artifact.id
                 info['youngest_artifact'] = '%s - %s' % (
                     youngest_artifact.name, youngest_artifact.artifact_type)
-                info['ebi_experiment'] = bool(prep.ebi_experiment_accessions)
+                info['ebi_experiment'] = bool(
```

Comment:
I am likely missing something, this is going to generate a list of "accessions", and if some are None, they will still be included in the resulting list, right? There's no filter being applied here. Otherwise, what's the goal of enumerating explicitly the values? If this is still the correct approach, maybe it would be shorter to use `bool(prep.ebi_experiment_accessions.values())`.